MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
You can still leave this as is, but I got MS to clarify the rules for internals in their style guide:

https://github.com/dotnet/corefx/wiki/Coding-style

> 3.We use  _camelCase  for internal and private members and use  readonly  where possible. Prefix instance fields with  _ , static fields with  s_  and thread static fields with  t_ .